app/views/user_mailer/analytics_report.html.erb

```diff
@@ -0,0 +1,193 @@
+<html>
+
+<head>
+  <meta content='text/html; charset=UTF-8' http-equiv='Content-Type' />
+  <style>
+    main {
+      max-width: 676px;
+      margin: auto;
+      font-family: Arial, Helvetica, sans-serif;
+    }
+
+    table {
+      width: 100%;
+      margin-bottom: 50px;
+      border-collapse: collapse;
+    }
+
+    th {
+      text-align: left;
+    }
+
+    .styled-table thead tr {
+      background-color: #212529;
+      color: #ffffff;
+      text-align: left;
+    }
+
+    .styled-table th,
+    .styled-table td {
+      padding: 12px 15px;
+      border-bottom: 1px solid rgb(222, 226, 230);
+    }
+
+    .users {
+      overflow: auto;
+    }
+  </style>
+</head>
+
+<body>
+  <main>
+    <h1>Periodic reports for visitor analytics</h1>
```

Comment:
the heading should be: 
"Analytics Report"
with subheading: 
"For period starting from #{start_date} #{end_date}"